mcp/cloudflare-worker/README.md

```
@@ -0,0 +1,185 @@
+     --header "X-Honcho-Assistant-Name:Claude"
+   ```
+
+   **For all available configuration options and custom headers, see the [Configuration Options](#configuration-options) section below.**
+
+### Configuration Options
+
+You can customize the behavior using HTTP headers:
+
+**Available Configuration:**
+- `apiKey`: Your Honcho API key
+- `baseUrl`: Custom Honcho API base URL (default: https://api.honcho.dev)
```

markdownlint-cli2 (0.17.2)

Bare URL used

(MD034, no-bare-urls)